### DISCUSSION TITLE
Why is pydantic pinned to v2.7.0?

### DISCUSSION THREAD

**Discussion by bencrouse** [567-labs/instructor#641] [Q&A]
Hey, I'd like to use this library in my project but I'm experiencing some dependency conflict because [this depends explicitly on pydantic v2.7.0](https://github.com/jxnl/instructor/blob/main/pyproject.toml#L14). 

I'm guessing there's a specific reason, but a quick search in issues/PRs didn't reveal anything. I see it was done with [this commit](https://github.com/jxnl/instructor/commit/aba1d73d08e44efa774c3c484341f111527ebd15) but I don't see a paper trail from that.

Anyone know what's up?

**Comment by jxnl** [comment]
for partial streaming

**Reply by jxnl** [reply]
There's older versions of the library that you can install. i think 1.1.0?

**Comment by jxnl** [comment]
definitly my bad on having poor PR hygene, but 2.7.0 allows for `partial=True` on their json loader

**Reply by bencrouse** [reply]
All good, thanks for responding! I was actually experiencing a conflict in the other direction, one of the libs I'm using depends on `^2.7.1` because of a patch that was present in that version of Pydantic.

**Reply by jxnl** [reply]
That shouldn't cause an issue with instructor, right?

**Reply by bencrouse** [reply]
Nah, I made a fork where [I bumped it](https://github.com/bencrouse/instructor/commit/b886aa42a92bd336078d20ed6f580dbd02ccb199) and it's been working fine. I'll open a PR, lemme know if there's anything I can do to help.

**Reply by bencrouse** [reply]
([PR opened](https://github.com/jxnl/instructor/pull/643))
